server/src/main/java/org/apache/iotdb/db/query/control/TracingManager.java

```diff
@@ -32,6 +32,9 @@
 public class TracingManager {
 
   private static final Logger logger = LoggerFactory.getLogger(TracingManager.class);
+  private static final String QUERY_ID = "Query Id: ";
+  private static final String NEW_LINE_QUERY_ID = "\nQuery Id: ";
```

Comment:
`NEW_LINE_QUERY_ID` looks weird. I think it's better to use `"\n" + QUERY_ID` in code below.